gateway-provider-security-jwt/src/main/java/org/apache/knox/gateway/provider/federation/jwt/JWTMessages.java

```diff
@@ -104,4 +104,11 @@ public interface JWTMessages {
 
   @Message( level = MessageLevel.WARN, text = "Invalid SSO cookie found! Cleaning up..." )
   void invalidSsoCookie();
+
+  @Message( level = MessageLevel.WARN, text = "User with SSO token {0} exceeded the configured idle timeout of {1} seconds." )
```

Comment:
Is it possible to include the principal name here for easier correlation to the user?

Reply:
Fixed.